src/pages/claim/index.vue

```diff
@@ -240,30 +240,26 @@ onBeforeMount(async () => {
           {{ configService.network.chainName }} {{ $t('liquidityIncentives') }}
         </h2>
 
-        <template v-if="networkHasNativeGauges">
```

Comment:
Could change this to `networkHasBalClaiming` instead of removing it, which checks for the minter being set as I assume we'll have networks that you can't claim BAL on in the future.

Reply:
hm, yeah, good catch. thx